lib/hex_web/web/templates/docs/mirrors.html.eex

```diff
@@ -15,7 +15,13 @@
 </p>
 
 <p>
-  <strong>Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+  <strong>Mix Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+</p>
+<p>
+  <strong>Rebar3 Example</strong>: <code>{rebar_packages_cdn, "https://s3.amazonaws.com/s3.hex.pm"}.</code>
+</p>
```

Comment:
Maybe we should drop the second example in that case?

We should add that it goes in `rebar.config` like you suggested and possibly add a link to rebar3 docs about configuration if you have it and if it fits on the page.